pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -82,6 +83,9 @@ var _ = rest.Creater(&REST{})
 var (
 	ForbiddenNames    = []string{"openshift", "kubernetes", "kube"}
 	ForbiddenPrefixes = []string{"openshift-", "kubernetes-", "kube-"}
+
+	defaultRoleBindingNames = bootstrappolicy.GetBootstrapServiceAccountProjectRoleBindingNames()
+	roleBindingKind         = rbac.Kind("RoleBinding")
```

Comment:
template can be old enough that it's openshift groupified and openshift non-groupified too.

Reply:
OK